### ISSUE TITLE
obs to Bitfocus connection

### ISSUE THREAD

**Issue by MPCSound** [bitfocus/companion-module-obs-studio#258]
**Describe the bug**
Obs Plugin for bitfocus will not connect to OBS.   Getting a time out error.  WebSocket setting are correct and I see the plugin connection in the WebSocket session but I get not response.   This was working about 2 weeks ago.  I have tried reinstalling OBS. Any Ideas? 

**Version Information:**

- Operating System:  Windows 11
- OBS Version:  30.0.2
- Companion Version: Raspberry Pi Beta: 3.3.0(3.3.0+6817).  I have also tried to run it off the PC and getting the same results

**Comment by bryce-seifert** [comment]
If you have the module logs open (the >_ icon next to the module name) do you see any errors when you stop start the connection?

**Comment by MPCSound** [comment]
This is all I get

debug: Only recent lines are shown here, nothing is persisted
error: Connection lost to OBS
info: Connected to OBS
debug: OBS Version: 30.0.2 // OBS Websocket Version: 5.3.4 // Platform: Windows 11 Version 22H2
debug: Request GetReplayBufferStatus failed (Error: Replay buffer is not available.)

**Comment by bryce-seifert** [comment]
@MPCSound That's strange. Do you have a scene collection .json I could perhaps test with? You can upload it [here](https://www.dropbox.com/request/Su4GDw6pRO87rqADDSsW)

**Comment by MPCSound** [comment]
Uploaded the file.   In companion it looks like a good connection with green checked mark and in WebSocket i see the connection but the session duration filed will run then stop.

**Comment by bryce-seifert** [comment]
Ok I just tested the collection and am seeing the same thing:
Companion seems just fine, but the Session Duration in the websocket UI seems to disappear right at a minute. I also tested this with http://obs-web.niek.tv and it does the same thing: session duration disappears at 1:00. So it seems like an obs-websocket or OBS bug.

Despite the OBS UI issue, everything seems to be working and connected with Companion. To confirm: Is that the case for you too?
